cfme/tests/containers/test_containers_default_project_replicators.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import CheckboxTable
+from cfme.containers.replicator import Replicator
+from itertools import product
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        (lambda: current_version() < "5.6") or (lambda provider: provider.version < 3.2)),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
+
+
+REPLICATORS_PROPERTIES_FIELDS = ['Name', 'Creation timestamp', 'Resource version',
+                                 'Requested pods', 'Current pods']
+REPLICATORS_RELATIONSHIPS_FIELDS = ['Containers Provider', 'Project', 'Pods', 'Nodes',
+                                    'My Company Tags']
+
+# CMP - 9531
+
+
+@pytest.mark.parametrize('replicators_properties_fields', product(REPLICATORS_PROPERTIES_FIELDS))
+def test_containers_replicators_properties(provider, replicators_properties_fields):
```

Comment:
Do not do the joins in first place. The parameters are strings, not arrays so it makes no sense.

Send me the list of collected tests for this file (--collect-only)